src/main/java/gregtech/loaders/recipe/VanillaOverrideRecipes.java

```diff
@@ -264,6 +266,7 @@ private static void redstoneRecipes() {
         );
 
         ModHandler.removeRecipeByName(new ResourceLocation("minecraft:daylight_detector"));
+        ModHandler.removeRecipeByName(new ResourceLocation("appliedenergistics2:misc/vanilla_daylight_detector"));
```

Comment:
Should these AE recipe removals be behind a modloaded check?

Reply:
It doesn't matter, they could but it makes no difference either way